.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "image": "mcr.microsoft.com/powershell/test-deps:ubuntu-22.04",
+    "features": {
+        "ghcr.io/devcontainers/features/github-cli:1": {},
```

Comment:
Do you also need the following entry for features?:

"ghcr.io/devcontainers/features/docker-in-docker:2": {},